enterprise/backend/test/metabase_enterprise/audit_db_test.clj

```diff
@@ -50,6 +51,18 @@
       (is (not= 0 (t2/count 'Card {:where [:= :database_id (audit-db/default-audit-db-id)]}))
           "Cards should be created for Audit DB when the content is there."))))
 
+(deftest audit-db-does-not-have-scheduled-syncs
+  (mt/test-drivers #{:postgres :h2 :mysql}
+    (with-audit-db-restoration
+      (is (= :metabase-enterprise.audit-db/installed (audit-db/ensure-audit-db-installed!)))
+      (let [sync-job-info (task/job-info "metabase.task.sync-and-analyze.job")
+            db-has-sync-job-trigger? (fn [db-id]
+                                       (contains?
+                                        (set (map #(-> % :data (get "db-id")) (:triggers sync-job-info)))
+                                        db-id))]
+        (is (db-has-sync-job-trigger? 1))
```

Comment:
What's the purpose of this assertion?

Reply:
To make sure we are looking at the right thing. But it's apparent in hindsight